bin/dcps_tests.lst

```diff
@@ -123,6 +123,7 @@ tests/DCPS/Reconnect/run_test.pl restart_pub: !DCPS_MIN !OPENDDS_SAFETY_PROFILE
 tests/DCPS/Reconnect/run_test.pl bp_timeout: !DCPS_MIN !OPENDDS_SAFETY_PROFILE !DDS_NO_OWNERSHIP_PROFILE
 tests/DCPS/Reconnect/run_test.pl sub_init_crash: !DCPS_MIN !OPENDDS_SAFETY_PROFILE !DDS_NO_OWNERSHIP_PROFILE
 tests/DCPS/Restart/run_test.pl: RTPS !OPENDDS_SAFETY_PROFILE
+tests/DCPS/Restart/run_test.pl --secure: RTPS !OPENDDS_SAFETY_PROFILE
```

Comment:
Should this be in `security_tests.lst`?

Reply:
It could be. I was following the lead of Messenger which is the only other test with optional security.